artemis-server/src/main/resources/schema/artemis-configuration.xsd

```diff
@@ -3937,11 +3937,29 @@
             </xsd:annotation>
          </xsd:element>
 
-         <xsd:element name="max-read-page-bytes" type="xsd:string" default="20M" maxOccurs="1" minOccurs="0">
+         <xsd:element name="prefetch-page-messages" type="xsd:int" default="20MB" maxOccurs="1" minOccurs="0">
+            <xsd:annotation>
+               <xsd:documentation>
+                  How many messages we are reading from paging and keeping ready in queues ready to deliver to consumers. Default: If not defined max-read-page-messages is used.
```

Comment:
This, coupled with max-page-read-size starting "How many messages are we allowed to read from page into the Queue each time." doesnt do that great a job of conveying the behaviour or relationship between the two.

Arguably max-read-page-messages needs updated to drop the "each time" to help clarify it being a limit on the total read in, while this one further allows limiting the amount read in at any given time to a smaller number.

```suggestion
                  How many messages the broker will read from paging and keep ready in queues ready to deliver to consumers. Default: If not defined max-read-page-messages is used.
```